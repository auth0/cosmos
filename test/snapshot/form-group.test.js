import React from 'react'
import renderer from 'react-test-renderer'
import { FormGroup } from '@auth0/cosmos'
import 'jest-styled-components'

describe('FormGroup', () => {
  it('renders with automation attribute', () => {
    const tree = renderer.create(<FormGroup />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
