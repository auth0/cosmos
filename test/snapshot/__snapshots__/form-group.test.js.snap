// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormGroup renders with automation attribute 1`] = `
<div
  className=""
  data-cosmos-key="form-group"
/>
`;
